Plugins~/Src/MeshSyncClientBlender/python/3.1.0/unity_mesh_sync.py

```diff
@@ -25,6 +25,8 @@
 # Called after export is finished
```

Comment:
we are missing all the versions before 3.1, is this intended?

Reply:
Yes, this is mostly for Bay's team, who are using later blender versions and I doubt anyone else will use this in the near future while the older versions are supported.